openprescribing/frontend/management/commands/preview_measure.py

```diff
@@ -19,7 +19,11 @@ def handle(self, measure_url, **options):
                 f"Importing measure preview failed for {measure_url}\n\n{e.message}"
             )
         measure_url = f"https://openprescribing.net/measure/{measure_id}/"
-        self.stdout.write(f"Measure can be previewed at:\n{measure_url}")
+        self.stdout.write(
+            f"Measure can be previewed at:\n{measure_url}\n\n"
+            f"When you've finished remember to delete the preview with:\n"
+            f"@ebmbot op delete_measure {measure_id}"
```

Comment:
Could this be `delete_preview_measure`?

Reply:
I went back and forth on this a bit, but maybe that is better actually. Do you think it would be sensible to make the ebmbot command `delete_preview_measure` but leave the Django command as is? I can sort of imagine us wanting to add a not-just-previews flag to that command for our own use later on.

Reply:
Leaving this as is, changing the ebmbot command would be fine.